committee-code-of-conduct/README.md

```diff
@@ -1,7 +1,40 @@
-# Kubernetes Code of Conduct Committee (CoCC)
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Code of Conduct Committee
 
 The Kubernetes Code of Conduct Committee (CoCC) is the body that is responsible for enforcing and maintaining the Kubernetes Code of Conduct.
 
+
+## Members
+
+* Jaice Singer Dumars (**[@jdumars](https://github.com/jdumars)**), Google
+* Paris Pittman (**[@parispittman](https://github.com/parispittman)**), Google
+* Carolyn Van Slyck (**[@carolynvs](https://github.com/carolynvs)**), Microsoft
+* Eric Paris (**[@eparis](https://github.com/eparis)**), Red Hat
+* Jennifer Rondeau (**[@bradamant3](https://github.com/bradamant3)**), VMware
+
+## Contact
+* Private Mailing List: conduct@kubernetes.io
+* [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/ug%2Fconduct)
+
+## GitHub Teams
+
+The below teams can be mentioned on issues and PRs in order to get attention from the right people.
+Note that the links to display team membership will only work if you are a member of the org.
+
+| Team Name | Details | Description |
+| --------- |:-------:| ----------- |
+| @kubernetes/code-of-conduct-committee | [link](https://github.com/orgs/kubernetes/teams/code-of-conduct-committee) | General Discussion |
+
+<!-- BEGIN CUSTOM CONTENT -->
+## Terms
```

Comment:
I wasn't sure where this information should go, so I put it in the custom content as "Terms". PTAL if this is ok.